pipeline/train/train.sh

```diff
@@ -1,11 +1,92 @@
 #!/bin/bash
+
 ##
 # Train a model.
 #
 
+##########################################################################
+# Slurm directives; see slurm documentation for what these parameters mean.
+# This script can also be run on the commandline outside of slurm.
+# In this case, slurm directives are simply ignored.
+#
+# Slurm directives start with #SBATCH. Do not remove the leading #.
+# It's part of the directive. Leading ## comments out the
+# respective slurm directive. All of the directives can also be
+# supplied on the command line when the slurm job is submitted.
+#
+# Uncomment and adjust the following to your own account settings. Of
+# course, these can also be provided with the sbatch call on the
+# command line.
+# Your account (-A) and the appropriate slurm partition (-p):
+##SBATCH -A <account>
```

Comment:
This is not a slurm job script. [This is](https://github.com/mozilla/firefox-translations-training/blob/main/profiles/slurm-csd3/jobscript.sh) a template of slurm jobscript that calls another script generated by snakemake and this another script calls this one at some point. So, these directives won't work.

Reply:
We appear to disagree what a slurm jobscript is. I'm used to using slurm on CSD3, and I invite you to tell me how my script is fundamentally different from the official documentation examples, e.g. at /usr/local/Cluster-Docs/SLURM/slurm_submit.wilkes2 on CSD3. Just because you prefer singularity containers via snakemake, it does not mean that any script that does not rely on that setup is an invalid slurm script. I'm willing  to learn and collaborate, but I do not like to be brushed off. As far as I can tell, this is a valid slurm script, although it may not work under your setup. As I said: willing to learn and collaborate, but there needs to be a spirit of collaboration, not scoring points.

Reply:
I welcome collaboration and my comment here is purely technical. Per my understanding, this is not a "slurm script" because it is not designed to be launched separately using `sbatch`. In the current architecture, it is launched by snakemake through multiple other scripts and I would call only the high level script that is actually launched with `sbatch` a "slurm script" (or jobscript) and only this script should be allowed to include slurm directives. Currently, it is [profiles/slurm-csd3/jpbscript.sh](https://github.com/mozilla/firefox-translations-training/blob/main/profiles/slurm-csd3/jobscript.sh).

Currently, everything inside the `pipeline` directory is not really designed to be launched without snakemake, even though it is technically possible. We can discuss this if you see value in running scripts separately locally or on slurm.